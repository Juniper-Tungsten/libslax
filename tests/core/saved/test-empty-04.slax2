version 1.1;

match / {
    var $user = "test";
    var $date = "now";
    
    <top> {
        comment "Added by user " _ $user _ " on " _ $date;
    }
}
