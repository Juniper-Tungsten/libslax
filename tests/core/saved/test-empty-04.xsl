<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
  <xsl:template match="/">
    <xsl:variable name="user" select="&quot;test&quot;"/>
    <xsl:variable name="date" select="&quot;now&quot;"/>
    <top>
      <xsl:comment>
        <xsl:value-of select="concat(&quot;Added by user &quot;, $user, &quot; on &quot;, $date)"/>
      </xsl:comment>
    </top>
  </xsl:template>
</xsl:stylesheet>
