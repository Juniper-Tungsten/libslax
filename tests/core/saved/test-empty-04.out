<?xml version="1.0"?>
<top>
  <!--Added by user test on now-->
</top>
